### PR TITLE
add mt_convert.py

### PR DESCRIPTION
**Pull request by wsor4035**
adds script from https://github.com/wsor4035/mt_convert. if merged, will commit to readme there with a link here and archive it

### CODE REVIEW

File: snippets/mt_convert.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        else:
+            depends.append(i)
+    
+    if depends != "":
+        conffile.write("depends = " + ", ".join(depends) + "\n")
+    if odepends != "":
+        conffile.write("depends = " + ", ".join(odepends) + "\n")
+    print("[mt_convert]: depends.txt converted")
+
+conffile.close()
```
Comment (Panquesito7) [inline snippets/mt_convert.py]: (Just to prevent issues) Missing an endline.
```suggestion
conffile.close()

```

**Review by BuckarooBanzay — APPROVED** [review]

**Comment by S-S-X** [comment]
~~Probably not a big thing but did those old files have multi line syntax like mod.conf? Or was it only added with mod.conf?
Just a description if it allowed multiple lines, dependencies should be fine anyway.~~

Well, forget it. Didn't really read the code first, it does already handle this 100%

File: snippets/mt_convert.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    print("invalid file path")
+    exit()
+elif os.path.exists(os.path.join(sys.argv[1], "mod.conf")):
+    print("mod.conf already exists!")
+    exit()
+
+descpath = os.path.join(sys.argv[1], "description.txt")
+deppath = os.path.join(sys.argv[1], "depends.txt")
+
+conffile = open(os.path.join(sys.argv[1], "mod.conf"), "a")
```
Comment (S-S-X) [inline snippets/mt_convert.py:23]: Possibly allow running without arguments to run conversion in current directory.

But then also argument against this: it's not that hard to write that dot yourself.

```suggestion
path = (sys.argv[1:2] or ["."])[0]
if not os.path.exists(path):
    print("invalid file path")
    exit()
elif os.path.exists(os.path.join(path, "mod.conf")):
    print("mod.conf already exists!")
    exit()

descpath = os.path.join(path, "description.txt")
deppath = os.path.join(path, "depends.txt")

conffile = open(os.path.join(path, "mod.conf"), "a")
```

**Review by S-S-X — APPROVED** [review]

File: snippets/mt_convert.py (hunk 1 above)
Comment (wsor4035) [reply snippets/mt_convert.py]: ....ok, whatever

**Pull request merged**